src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/InstallPrePushHookTask.kt

```diff
@@ -23,38 +26,38 @@ open class InstallPrePushHookTask : DefaultTask() {
                 mkdir()
             }
         }
-        logger.info("hookDir: $hookDir")
+        logger.quiet("hookDir: $hookDir")
 
-        val prePushHookFile = File(hookDir, "/pre-push").apply {
+        val hookFile = File(hookDir, "/$hookFile").apply {
             if (!exists()) {
                 logger.info("Creating $this anew")
                 createNewFile()
                 setExecutable(true)
             }
         }
 
-        if (prePushHookFile.length() == 0L) {
+        if (hookFile.length() == 0L) {
             logger.info("Writing hook to empty file")
-            prePushHookFile.writeText(generateHook(gradleCommand, addShebang = true))
+            hookFile.writeText(generateHook(gradleCommand, addShebang = true))
         } else {
-            val prePushHookFileContent = prePushHookFile.readText()
-            val startIndex = prePushHookFileContent.indexOf(startHook)
+            val hookFileContent = hookFile.readText()
+            val startIndex = hookFileContent.indexOf(startHook)
             if (startIndex == -1) {
                 logger.info("Appending hook to end of existing non-empty file")
-                prePushHookFile.appendText(generateHook(gradleCommand))
+                hookFile.appendText(generateHook(gradleCommand))
             } else {
                 logger.info("Updating existing kotlinter-installed hook")
-                val endIndex = prePushHookFileContent.indexOf(endHook)
-                val newPrePushHookFileContent = prePushHookFileContent.replaceRange(
+                val endIndex = hookFileContent.indexOf(endHook)
+                val newHookFileContent = hookFileContent.replaceRange(
                     startIndex,
                     endIndex,
                     generateHook(gradleCommand, includeEndHook = false)
                 )
-                prePushHookFile.writeText(newPrePushHookFileContent)
+                hookFile.writeText(newHookFileContent)
```

Comment:
Should the two different hooks be different content? Intuitively I might expect a pre-commit hook to simply `formatKotlin` and continue the commit, aborting only if auto-formatting is not possible.
The rationale is a pre-push hook is a backstop to prevent wasting a build, but you chose to have your workflow otherwise unaltered. A pre-commit hook feels more like a "just quietly take care of it" situation. What do you think?

Reply:
That's a reasonable distinction to me, updates shortly